cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -549,23 +545,29 @@ describe('Dashboards', () => {
     })
   })
 
-  // Skipping until https://github.com/influxdata/ui/issues/2864 is resolved by Bonitoo
-  it.skip('creates a dashboard and downloads JSON', () => {
+  before(() =>
+    cy.exec('rm cypress/downloads/*', {
+      log: true,
+      failOnNonZeroExit: false,
+    })
+  )
+
+  it('creates a dashboard and downloads JSON', () => {
     cy.get('@org').then(({id: orgID}: Organization) => {
       cy.createDashboard(orgID).then(({body}) => {
         cy.fixture('routes').then(({orgs}) => {
           cy.visit(`${orgs}/${orgID}/dashboards/${body.id}`)
           cy.getByTestID('tree-nav')
           cy.getByTestID('nav-item-dashboards').click()
           cy.getByTestID('dashboard-card--name').click()
-          cy.getByTestID('page-title').type('dashboard') // dashboard name added to prevent failure due to downloading JSON with a different name
+          cy.getByTestID('page-title').type('dshboard') // dashboard name added to prevent failure due to downloading JSON with a different name
```

Comment:
the `dshboard` name caused issues last time for OSS, can we just name it `dashboard` please? See where I fixed it here: https://github.com/influxdata/ui/pull/2808